scripts/release-post.js

```diff
@@ -33,15 +33,14 @@ function sendRequest (opts) {
       headers: { 'User-Agent': 'nodejs.org release blog post script' }
     }, opts)
 
-    request(options, (err, res, body) => {
-      if (err) {
-        return reject(new Error(`Error requesting URL ${options.url}: ${err.message}`))
+    const { url, ...gotOptions } = options
```

Comment:
Is the destructuring really needed here?

Reply:
It isn't. I was just trying to be safe with the options if url was passed in twice. I took it out in another commit, along with a rebase to master :). Let me know what you think!